extensions/amp-autocomplete/0.1/amp-autocomplete.css

```diff
@@ -13,3 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-autocomplete {
+  position: relative;
```

Comment:
This needs to be in `amp.css` and under `amp-autocomplete` selector since it changes `display`.
Keeping it here and tied to a class that gets added in layoutCallback would cause layout change (possible jump) when extension loads / gets layout.